### DISCUSSION TITLE
Adding custom HTML & CSS to the Listen Along page

### DISCUSSION THREAD

**Discussion by tootcrazu** [Taiko2k/Tauon#1183] [General]
I was looking at adding some custom formatting to the listen along page for fun, but it looks like it won't recognise any files outside of the ones that started there to begin with?
![image](https://github.com/Taiko2k/Tauon/assets/139095756/19bb208f-542f-4d1a-86ea-1d168a88ee78)
(excuse my poor mouse handwriting)

I've checked, and the permissions and everything seem to be the same between the two images, and I can edit logo-bg.png and the changes will show, it just won't show any other images I try to add...

**Comment by Taiko2k** [comment]
Currently the assets are essentially whitelisted here Ref: https://github.com/Taiko2k/Tauon/blob/8ddaaf24c7c3302e67f886b2e068490ff3850042/t_modules/t_webserve.py#L125
